Source/carthage/Bootstrap.swift

```diff
@@ -26,7 +26,19 @@ public struct BootstrapCommand: CommandType {
 					carthage.println(formatting.bullets + "No Cartfile.resolved found, updating dependencies")
 					return project.updateDependencies(shouldCheckout: options.checkoutAfterUpdate)
 				}
-
+				
+				if let depsToUpdate = options.dependenciesToUpdate {
+					let resolvedCartfileContents = try! String(contentsOfURL: project.resolvedCartfileURL, encoding: NSUTF8StringEncoding)
+					let resolvedCartfile = ResolvedCartfile.fromString(resolvedCartfileContents).value!
```

Comment:
I was aware of that actually but I'm new enough to ReactiveCocoa that I quite honestly didn't know how to use the signal producer based flow that would introduce into this existing command handler. Would need your help to show how to do this, am afraid.